pulsar-client/src/main/java/org/apache/pulsar/client/impl/AbstractBatchMessageContainer.java

```diff
@@ -42,6 +42,7 @@ public abstract class AbstractBatchMessageContainer implements BatchMessageConta
     protected int maxBytesInBatch;
     protected int numMessagesInBatch = 0;
     protected long currentBatchSizeBytes = 0;
+    protected int batchAllocatedSize = 0;
```

Comment:
Mirror comment: `batchAllocatedSizeBytes ` is better, like `currentBatchSizeBytes`

Reply:
Nice suggestion. Updated